tests/JunitFormatterTest.php

```diff
@@ -168,7 +169,7 @@ private function prettyPrintXml(string $inputXml): string
     {
         $dom = new DOMDocument();
         $dom->preserveWhiteSpace = false;
-        $dom->formatOutput = true;
+        $dom->formatOutput = true;  // always in human-readable format
```

Comment:
I think we can now compare it 1:1 without altering the real output. It should just match the passed expected string.